voice/src/commonMain/kotlin/VoiceConnection.kt

```diff
@@ -58,7 +58,7 @@ public class VoiceConnection(
     connectionDetachDuration: Duration
 ) {
     public val scope: CoroutineScope =
-        CoroutineScope(SupervisorJob() + CoroutineName("kord-voice-connection[${data.guildId.value}]"))
+        CoroutineScope(Dispatchers.Default + SupervisorJob() + CoroutineName("kord-voice-connection[${data.guildId.value}]"))
```

Comment:
shouldn't this use the Kord dispatcher?

Reply:
voice doesn't depend on core. If the scope was able to be configured it could be overridden in the core-voice module.

Reply:
Then that's what we should do